src/App.tsx

```diff
@@ -15,36 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  default = '',
+  byAlphabet = 'alphabet',
+  byLength = 'length',
+}
+
+function getPreparedGoods(
+  goods: string[],
+  sortField: SortType,
+  reversed: boolean,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.byAlphabet:
+          return good1.localeCompare(good2);
+        case SortType.byLength:
+          return good1.length - good2.length;
+        default:
+          return 2;
```

Comment:
Returning `2` in the default case of the sort function is unconventional and can lead to unexpected behavior. It's more common to return `0` to indicate that there is no difference between the two items being compared.